wls-gui-wahllokalsystem/src/components/ergebnisermittlung/TheErgebnisermittlungStimmzettelumschlaegeCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title>Zählen der Stimmzettel</v-card-title>
+      <v-card-text class="pb-0 pt-2">
+        <v-form
+          ref="anzahlStimmzettelForm"
+          v-model="anzahlStimmzettelValidForm"
+        >
+          <div class="d-flex flex-wrap justify-start">
+            <div>
+              <v-number-input
+                v-model="wahl.stimmzettelumschlaege.anzahlWaehler"
+                class="mr-4"
+                :rules="[REQUIRED, MIN_NUMBER(0), MAX_NUMBER(9999)]"
+                min-width="30rem"
```

Comment:
finde 20 ausreichend, aber vielleicht sollten wir das auch mal besprechen ob wir das zentral festlegen für alle inputs
```suggestion
                min-width="20rem"
```